src/align.py

```diff
@@ -64,7 +64,7 @@ def edits(x: str, y: str) -> str:
     
     for i in range(len(x)):
 
-        if x[i] in 'ATGC' and y[i] in 'ATGC':
+        if x[i] in 'ATGCatgc' and y[i] in 'ATGCatgc':
```

Comment:
Testing

```python
if x[i] != '-' and y[j] != '-':
   ...
```

would be faster. Right now, you are doing a linear search for each pair of characters to check if the are in the DNA alphabet.